### DISCUSSION TITLE
腾讯服务器lcu接口被禁， 一个新的解决方案

### DISCUSSION THREAD

**Discussion by SYJun404** [cangzhang/ChampR#233] [General]
# 腾讯服务器lcu接口被禁， 一个新的解决方案

LOL的驱动做了限制，只允许白名单的程序访问

程序文件名改成英雄联盟目录里程序，我用的是RiotClientCrashHandler  【frank.exe --->RiotClientCrashHandler.exe 】（目的是变成白名单）

好像这样就可以了， 我自己的电脑可以正常访问lcu接口

**Comment by SYJun404** [comment]
好像又不行了....

**Comment by arrebole** [comment]
@Java-S12138 外服的有没有被禁？还是只有国服的接口被禁了?

**Reply by SYJun404** [reply]
只有国服被禁的

**Comment by arrebole** [comment]
@Java-S12138 我试过进一次测试服的下一次登录可以正常，没有账号也没关系，提示没有资格然后退出

**Comment by weituotian** [comment]
好~(￣▽￣)~*

**Comment by weituotian** [comment]
现在测试服好像都不能用lcuapi了，真的绝了

**Reply by arrebole** [reply]
不知道改动了什么，国服又正常了，我试了现在可以连接lcuapi了和以前一样
